pkg/machinery/kernel/kernel.go

```diff
@@ -32,9 +32,6 @@ func DefaultArgs(quirks quirks.Quirks) []string {
 		"nvme_core.io_timeout=4294967295",
 		// Disable rate limited printk
 		"printk.devkmsg=on",
-		"ima_template=ima-ng",
```

Comment:
I think this should be a `quirk` so that we don't change old Talos versions kernel args?